.github/workflows/main.yml

```diff
@@ -22,6 +22,6 @@ jobs:
 
       - run: pip install --upgrade setuptools
       - run: pip install -r requirements.txt
-      - run: pip install .
+      - run: pip install -e .
```

Comment:
This avoids errors around PEP-517 compliance and is what `ion-python` does, so it did it here as well.